app/models/automated_checks/claim_verifiers/identity.rb

```diff
@@ -17,23 +17,24 @@ def initialize(
       def perform
         return unless awaiting_task?(TASK_NAME)
 
-        # Order of matching matters so that subsequent conditions in methods fall through to execute the right thing
-        auto_pass || no_match || partial_match || complete_match
+        if claim.identity_confirmed_with_onelogin?
+          one_login
+        else
+          # Order of matching matters so that subsequent conditions in methods fall through to execute the right thing
+          no_match || partial_match || complete_match
+        end
       end
 
       private
 
       attr_accessor :admin_user, :claim
       attr_reader :dqt_teacher_status
 
-      def auto_pass
-        case claim.policy.auto_pass_identity_confirmation_task(claim)
-        when :pass
+      def one_login
+        if claim.onelogin_idv_full_name.downcase == claim.full_name.downcase && claim.onelogin_idv_date_of_birth == claim.date_of_birth
```

Comment:
Probably one for product, but if one login only returns first + last name should this comparison be against `Claim#first_name` and `Claim#surname` rather than `Claim#full_name` which includes middle name?